helpick-service/src/main/java/com/infoshare/domain/NeedRequest.java

```diff
@@ -2,93 +2,96 @@
 
 import java.util.Date;
 import java.util.Objects;
+import java.util.UUID;
 
 public class NeedRequest {
 
-    private TypeOfHelp typeOfHelp;
-    private HelpStatuses helpStatus;
-    private Date statusChange;
-    private PersonInNeed personInNeed;
+  private TypeOfHelp typeOfHelp;
+  private HelpStatuses helpStatus;
+  private Date statusChange;
+  private PersonInNeed personInNeed;
+  private final UUID uuid;
 
 
-    public NeedRequest() {
-    }
+  public NeedRequest(TypeOfHelp typeOfHelp, HelpStatuses helpStatus, Date statusChange, PersonInNeed personInNeed,
+      UUID uuid) {
+    this.typeOfHelp = typeOfHelp;
+    this.helpStatus = helpStatus;
+    this.statusChange = statusChange;
+    this.personInNeed = personInNeed;
+    this.uuid = uuid;
+  }
 
-    public NeedRequest(TypeOfHelp typeOfHelp, HelpStatuses helpStatus, Date statusChange, PersonInNeed personInNeed) {
-        this.typeOfHelp = typeOfHelp;
-        this.helpStatus = helpStatus;
-        this.statusChange = statusChange;
-        this.personInNeed = personInNeed;
-    }
 
-    public NeedRequest(TypeOfHelp typeOfHelp, PersonInNeed personInNeed) {
-        this.typeOfHelp = typeOfHelp;
-        this.helpStatus = HelpStatuses.NEW;
-        this.statusChange = new Date();
-        this.personInNeed = personInNeed;
-    }
+  public static NeedRequest createNeedRequest(TypeOfHelp typeOfHelp,
+      PersonInNeed personInNeed) {
```

Comment:
Może wystarczy samo `create`?
```suggestion
    public static NeedRequest create(TypeOfHelp typeOfHelp, PersonInNeed personInNeed) {
```